pkg/base-dev/sur/hood.hoon

```diff
@@ -65,6 +65,8 @@
   =+  .^  raz=(list vat)
           %gx  /(scot %p our)/hood/(scot %da now)/kiln/vats/noun
       ==
+  %+  weld
```

Comment:
originally I had this the other way so it needed to be a weld but now it can just be a `:-`